Pdrapp

deoyani opened the pull request:
@RayPlante Please check some changes after our last review.We still need to discuss many things with respect to landing page.

RayPlante commented:
Sorry for losing track of this one; doing a test now.  If you would, could you add a little more commentary about what is included in this PR?  thanks!

deoyani commented:
- Updated the UI by changing navigation panels.
- Removed the unused links and updated the contents.
- Suggestions made by Ray and Gretchen are incorporated (80%). There are few changes need to be complete.
- displaying Metadata as a tree format is still under development.   
- reorganized various components
